pom.xml

```diff
@@ -44,20 +44,24 @@
         please contact the sales department at sales@jahia.com.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.jahia.modules</groupId>
         <artifactId>jahia-modules</artifactId>
-        <version>7.2.3.2</version>
+        <version>7.5.0.0-SNAPSHOT</version>
```

Comment:
why 7.5-SN ? this version will never be released, we'll release 8 just after 7.3.4.x 
so maybe we can put 7.3.4.0-SN and the other commits from jahia-ee and jahia-private should be cherry picked into the 7.3.4.X-BRANCH